R/utils.R

```diff
@@ -100,6 +100,33 @@ r_files <- function(path) {
   sort_c(dir(file.path(path, "R"), "[.Rr]$", full.names = TRUE))
 }
 
+ignore_files <- function(path, rfiles) {
```

Comment:
I think the arguments should be swapped: the primary thing being modified is the vector of paths to R files